core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java

```diff
@@ -351,6 +358,13 @@ private boolean isCMRunning(ClusterApi connector) {
         return Optional.empty();
     }
 
+    private Set<String> getComputeHostGroups(Cluster cluster) {
+        String blueprintText = cluster.getBlueprint().getBlueprintText();
+        CmTemplateProcessor blueprintProcessor = cmTemplateProcessorFactory.get(blueprintText);
+        Versioned blueprintVersion = () -> blueprintProcessor.getVersion().get();
+        return blueprintProcessor.getHostGroupBasedServiceAttributes(blueprintVersion).keySet();
```

Comment:
Wouldn't the keySet include all hostGroups containing YARN? The filtering needs to be based on the values and the attributes set there?

Reply:
Great catch, fixed it.